nni/runtime/protocol.py

```diff
@@ -7,26 +7,12 @@
 
 from .tuner_command_channel.command_type import CommandType
 from .tuner_command_channel import legacy
-from .tuner_command_channel import shim
```

Comment:
This file is abandoned?

Reply:
Yes. The migration is nearly complete.

Reply:
Well, in that case, I suggest adding a deprecated warning when someone tries to import this file.

Reply:
Removed.